PubNub/Data/PNAES.m

```diff
@@ -6,6 +6,9 @@
 #import "PNAES.h"
 #import <CommonCrypto/CommonCryptor.h>
 #import <CommonCrypto/CommonHMAC.h>
+#ifdef PGDROID
+//#import <CommonCrypto/CommonCryptoError.h>
```

Comment:
Actually we should just remove this.  Done in 7ef9eb4b52c7588239d5381e667b748e43fe78b5